### PR TITLE
Issue 411 doc improvements

### LINKED ISSUE #411: Review existing docs and fix
Update the existing documents and fill in holes.

### LINKED ISSUE #412: Add in addendum documentation on how to deploy the Siddhi AE to K8s.
Siddhi is running as a Java app in CI but can also run in K8s. This purpose of this task is to cherry pick from the k8s-siddhi-operator-ae branch what makes sense as we will not be automating this functionality on EC2 at this time.

### PR DESCRIPTION
**Pull request by spisarski**
#### What does this PR do?
Fixes #411 & Fixes #412
Updated, improved, and tested processes identified all docs to reflect the current state of the repository
Added more links in MD docs
Included examples and CRDs for setting up and running a TPS AE running on Kubernetes
#### Do you have any concerns with this PR?
no
#### How can the reviewer verify this PR?
Ensure the docs are accurate...
#### Any background context you want to provide?
We are trying to ready this repository for mothballs so it will be easier to reconstitute down-the-road
#### Questions:
- Have you connected this PR to the issue it resolves? yes
- Does the documentation need an update? yes
- Does this add new dependencies? no
- Have you added unit or functional tests for this PR? no

### CODE REVIEW

**Comment by spisarski** [comment]
Run Tofino lab trial build

**Comment by spisarski** [comment]
Run Tofino aggregate build

**Review by chriscorcimiglia — CHANGES_REQUESTED** [review]
Looking good. I made some comments that you should be able to address pretty easily. Thanks for pulling all of this together!

File: docs/ae/SIDDHI_AE_SETUP.md [hunk 1]
```diff
@@ -21,30 +21,55 @@ Kubernetes where the siddhi-operator has been installed.
...
-  project: https://github.com/cablelabs/siddhi-map-p4-trpt.git
-- Open shell to the cloned directory (e.g. cd ~/siddhi-map-p4-trpt)
-- Execute
+### Java application
+The top-level Ansible playbook performing the actions listed below can be found
+[here](../../playbooks/siddhi/setup_siddhi_maven.yml)
+
+- Install Java 8+ - [playbook](../../playbooks/siddhi/setup_jdk.yml) (setting up OpenJDK 11.0.11)
+- Install and start Kafka to port 9092 - [playbook](../../playbooks/siddhi/setup_kafka.yml)
+  ([installed based on these instructions](https://www.digitalocean.com/community/tutorials/how-to-install-apache-kafka-on-ubuntu-20-04)
```
[the remark this answers is not in the document]
Comment (spisarski) [reply docs/ae/SIDDHI_AE_SETUP.md:32]: Actually added the left paren on accident

**Review by chriscorcimiglia — APPROVED** [review]

**Pull request merged**